### PR TITLE
After starting app margin needs to be done between header message and content #36

### PR DESCRIPTION
**Pull request by kapilG0**
Closes: #36 
- **Title** : Contributor
- **Name:** Kapil Gupta
- **Idenitfy yourself:** SSOC,GSSOC

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

### Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/README.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/CODE_OF_CONDUCT.md) of this project.
- [ ] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
image

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
image
Still shows same issue.

**Comment by kapilG0** [comment]
> image Still shows same issue.

Ok let me check again

**Comment by kapilG0** [comment]
> > image Still shows same issue.
> 
> Ok let me check again

Hello @Avdhesh-Varshney  please check now i have fixed that and commit my code again in same pr

File: src/App.jsx [hunk 1]
```diff
@@ -41,9 +41,9 @@ function App() {
     setAlert({
       msg: message
     })
-    setTimeout(() => {
-      setAlert(null);
-    }, 3500);
+    // setTimeout(() => {
```
Comment (Avdhesh-Varshney) [inline src/App.jsx:44]: don't remove this timeout.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@kapilG0 try to solve it with the presence of timeout operation.
And the change the title of the pr same as issue

**Comment by kapilG0** [comment]
> @kapilG0 try to solve it with the presence of timeout operation. And the change the title of the pr same as issue

Hello @Avdhesh-Varshney thank you for your response i have added that and update header

**Review by Avdhesh-Varshney — APPROVED** [review]
@kapilG0 PR Approved 🎉
But for next time, don't do un-necessary changes without discussing.

**Pull request merged**